useScroll/Scrollbar.js

```diff
@@ -126,6 +151,24 @@ ScrollbarBase.propTypes = /** @lends agate/useScroll.Scrollbar.prototype */ {
 	 */
 	css: PropTypes.object,
 
+	/**
+	 * Specifies to reflect scrollbar's disabled property to the paging controls.
+	 * When it is `true`, both prev/next buttons are going to be disabled.
+	 *
+	 * @type {Boolean}
+	 * @public
```

Comment:
Please add '@default false' here.

Reply:
@0x64 Oh I've remembered that we don't need to do this. Refer to https://github.com/enyojs/sandstone/pull/483#discussion_r442575276.

Reply:
@MikyungKim Right. Thanks!

Reply:
@jeonghee27 Sorry, let's get rid out of it.